### ISSUE TITLE
Group Normalization

### ISSUE THREAD

**Issue by khoshsirat-udel** [mapillary/inplace_abn#127]
Hi,
Is it possible to add support for Group Normalization (normalization over groups instead of batches)?
https://arxiv.org/abs/1803.08494
torch.nn.GroupNorm

**Comment by ducksoup** [comment]
This is not a trivial task, unfortunately. It has been on our radar for a while, but we can't promise anything for now.
